src/data/Graph.ts

```diff
@@ -333,6 +333,9 @@ class GraphNode {
     // Used in traverse of Graph
     __visited: boolean;
 
+    // NOTE: only use in circular layout
+    private _fixed: boolean = false;
```

Comment:
I think there is a better place to store the `fixed` status rather than the very basic graph structure. In fact, `series-graph.node` has `fixed` option. But currently it's only for force layout.

Reply:
Currently store in the `fixed` option of nodeModel